nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -233,6 +233,10 @@ def get_ignored_metatypes(model_type: ModelType, device: TargetDevice) -> List[O
                 om.ONNXSqrtMetatype,
                 om.ONNXReciprocalMetatype,
                 om.ONNXBatchNormMetatype,
+                # Сomparison operations
+                om.ONNXGreaterMetatype,
+                om.ONNXLessMetatype,
+                om.ONNXEqualMetatype,
```

Comment:
Why only for `Transformer` models? Is there any reason we can't remove them from the configuration?

Reply:
It's required wide validation. I filed a ticket to do it 144218.

Reply:
Was there any validation for `Transformer` models at least?

Reply:
run post_training_quantization: 402